 # /** ------ MIL测试车道线信息描述包 ------ */

uint64  u8RoadId
float64 curvVert
float32 u4StartDx
float32 u4Width
float32 u4ViewRangeStart
float32 u4ViewRangeEnd
float32 u4MeasuredVREnd
float32 u4C0
float32 u4C1
float32 u4C2
float32 u4C3
uint16  u2LaneChange
int8    u1LaneId
uint8   u1Quality
uint8   u1Type
uint8   u1Color
uint8   u1LeftCrossing
uint8   u1RightCrossing